languages/kotlin/exercises/concept/annalyns-infiltration/.meta/config.json

```diff
@@ -0,0 +1,9 @@
+{
+  "authors": [
+    {
+      "github_username": "ErikSchierboom",
+      "exercism_username": "ErikSchierboom"
```

Comment:
You should put your own name here, as _you_ are the author of this exercise (i.e. you've done the work to create it). I will be credited indirectly through the `forked_from` exercise, of which I'm the author.

Reply:
Got it. Will change